README.md

```diff
@@ -23,10 +23,12 @@ The following text provides a step-by-step guide on how to import [Jackson BOM](
 
 Add the `sbt-bom` plugin to `plugins.sbt` file.
 ```scala
-addSbtPlugin("com.here.platform" % "sbt-bom" % "1.0.14")
+addSbtPlugin("com.here.platform" % "sbt-bom" % "x.y.z") // Check latest version below
 ```
 
-#### Add Maven central repository in your resolvers
+Latest version: ![Sonatype Central](https://maven-badges.sml.io/sonatype-central/com.here.platform/sbt-bom_2.12_1.0/badge.svg?style=social)
+
+#### (Optional) Add Maven central repository in your resolvers
```

Comment:
Let's keep this section name as is. My teammate will update usage documentation soon